test/com/dept24c/integration/integration_test.cljc

```diff
@@ -249,12 +249,18 @@
                unsub! (vivo/subscribe! vc sub-map nil #(ca/put! state-ch %)
                                        "test")
                _ (is (= {'subject-id nil} (au/<? state-ch)))
+               login-fail (au/<? (vivo/<log-in!
```

Comment:
While the correct `{'subject-id nil}` gets put on the `state-ch` the return value is actually the empty map. In other words `login-fail` is `{}`. I made a brief attempt to find out why but I'm out of time for the day. It's like something between the server returning `{:subject-id nil :token nil}` and `login-fail` is calling `(remove nil? ...)` but nothing obvious came up.